src/hooks/constants/lib/fetchConstants.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants } from '../../../generated/graphql';
+import { fetchConstants } from './fetchConstants';
+import {
+  getMockApiPromise,
+  mockedRepayFee,
+} from '../../polkadotJs/tests/mockUsePolkadotJsContext';
+
+describe('fetch chain constants', () => {
```

Comment:
please name the top level describe as the file name you're testing, e.g. `fetchConstants`